File: bubblesort/bubblesort.py
def bubblesort(array): 
        listlen = len(array)    
        swapped = True

        while swapped:
                swapped = False
                for i in range(listlen - 1):
                        if array[i] > array[i + 1]:
                                temp = array[i] # save the current element of the array in a temporary element
                                array[i] = array[i + 1] # replace the position of the element with the next element of the list  
                                array[i + 1] = temp # set the next element as the element of the current position
                                swapped = True

        return array



array = [23,34,1,267,23,189,13,3,2,136,5,23]
print(bubblesort(array))

File: Algorithims/Sort/bubblesort/bubblesort.js
function bubbleSort(arr){
    let arralen = arr.length; // array length
    let swapped; 

    do{
        swapped = false;
        
        for( let i = 0; i < arralen; i++){ // loop through the array

            if(arr[i] > arr[i+1]){ // if the current element in the array is greater than the next element then do the following 
                let temp = arr[i]; // set the temp value as  the current element
                arr[i] = arr[i + 1]; // switch the current element position with the next element
                arr[i + 1] = temp; // set the next element as the current element which was stored in the temp folder 
                swapped = true; 
                console.log(swapped)
            }
        }

    }while(swapped)

    return arr;
}

console.log(bubbleSort([1,34,5,6,45,23,67,878,2234,5,2,4,11,3]))

// NOTES
/* 
    The algorithim is ineffcient since as the input increases more space and time will be required to sort the array
*/